pypeit/spectrographs/p200_dbsp.py

```diff
@@ -95,7 +95,7 @@ def compound_meta(self, headarr: List[fits.Header], meta_key: str):
             return Time(headarr[0]['UTSHUT']).mjd
         elif meta_key == 'dispangle':
             try:
-                return Angle(headarr[0]['ANGLE']).deg
+                return Angle(headarr[0]['ANGLE'].replace('Deg', 'deg')).deg
```

Comment:
I think that using `headarr[0]['ANGLE'].lower()` might make the code more robust (for example if the DBSP controller writes `"20 Deg 37 Min"` in the `ANGLE` field).

Reply:
I agree, can someone make that change please, then I'll approve.

Reply:
done. waiting for CI

Reply:
Oops, missed this thread until now, but thanks for taking care of it, @finagle29 !